### PR TITLE
New package: bacula-common-7.4.7

### PR DESCRIPTION
**Pull request by mfulz**
Bacula stuff: fd, sd, dir, and alternatives for the db backend.

### CODE REVIEW

File: srcpkgs/bacula-common/template [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+do_install() {
+	vinstall "scripts/${pkgname%-*}_config" 750 "etc/${pkgname%-*}/scripts/"
+	vinstall "scripts/btraceback.gdb" 644 "etc/${pkgname%-*}/scripts/"
+	vinstall "scripts/btraceback.dbx" 644 "etc/${pkgname%-*}/scripts/"
+	vinstall "scripts/btraceback.mdb" 644 "etc/${pkgname%-*}/scripts/"
+	vinstall "scripts/btraceback" 750 "usr/bin/"
+	vinstall "src/lib/.libs/libbac-${version}.so" 755 "usr/lib/"
+	vinstall "src/lib/.libs/libbaccfg-${version}.so" 755 "usr/lib/"
+	vinstall "src/findlib/.libs/libbacfind-${version}.so" 755 "usr/lib/"
+	vman "manpages/btraceback.8"
```
Comment (Vaelatern) [inline srcpkgs/bacula-common/template]: Does the makefile do none of this?

File: srcpkgs/bacula-fd/template [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+makedepends="libressl-devel zlib-devel"
+depends="bacula-common"
+wrksrc="${pkgname%-*}-${version}"
+conf_files="/etc/${pkgname%-*}/${pkgname}.conf"
+
+do_install() {
+	vinstall "src/filed/${pkgname}.conf" 640 "etc/${pkgname%-*}/"
+	vinstall "src/filed/.libs/${pkgname}" 750 "usr/bin/"
+	vinstall "src/plugins/fd/.libs/bpipe-fd.so" 750 "usr/lib/"
+	vman "manpages/${pkgname}.8"
```
Comment (Vaelatern) [inline srcpkgs/bacula-fd/template]: Same here

File: srcpkgs/bacula-common/template [hunk 3]
```diff
@@ -0,0 +1,25 @@
+pkgname="bacula-common"
```
Comment (Vaelatern) [inline srcpkgs/bacula-common/template]: start with a comment as per most every other template

File: srcpkgs/bacula-fd/template [hunk 4]
```diff
@@ -0,0 +1,22 @@
+pkgname="bacula-fd"
```
Comment (Vaelatern) [inline srcpkgs/bacula-fd/template]: And here

File: srcpkgs/bacula-common/template (hunk 3 above)
Comment (mfulz) [inline srcpkgs/bacula-common/template]: OK will do this

File: srcpkgs/bacula-common/template (hunk 1 above)
Comment (mfulz) [inline srcpkgs/bacula-common/template]: It does but if I just usw the makefile, I'll be unable to split the package in the different parts: bacula-fd, bacula-sd, bacula-dir-mysql, etc. 
I would have then just one big package which doesn't make much sense

File: srcpkgs/bacula-fd/template (hunk 4 above)
Comment (mfulz) [inline srcpkgs/bacula-fd/template]: Ok

File: srcpkgs/bacula-fd/template (hunk 2 above)
Comment (mfulz) [inline srcpkgs/bacula-fd/template]: See the answer above

**Comment by mfulz** [comment]
Fixed it now :)

**Comment by Vaelatern** [comment]
Please shrink your musl patch. Remove the plain whitespace fixes, and other fixes that have no actual effect. Will make the patch easier to review.
Also please rebase on master.

**Comment by mfulz** [comment]
Done

**Comment by mfulz** [comment]
Pushed to version 7.4.5 and added sv scripts

**Pull request closed without merging**